apps/web/pages/settings/my-account/out-of-office/index.tsx

```diff
@@ -39,18 +40,58 @@ export type BookingRedirectForm = {
   toTeamUserId: number | null;
   reasonId: number;
   notes?: string;
+  uuid?: string | null;
 };
 
-const CreateOutOfOfficeEntryModal = ({
+export type outOfOfficeEntryData = {
+  uuid: string;
+  selectedReason: number | null;
+  profileRedirect: boolean;
+  selectedMember: number | null;
+  notes: string | null;
+  start: Date;
+  end: Date;
+};
```

Comment:
No need to create another type. You can use `BookingRedirectForm` type for passing around data, which can then be directly used in defaultValues removing useEffect